src/main/java/org/opensearch/security/configuration/DlsFlsFilterLeafReader.java

```diff
@@ -1229,11 +1236,13 @@ private static class MaskedTermsEnum extends TermsEnum {
 
         private final TermsEnum delegate;
         private final MaskedField mf;
+        private final String algorithmDefault;
 
-        public MaskedTermsEnum(TermsEnum delegate, MaskedField mf) {
+        public MaskedTermsEnum(TermsEnum delegate, MaskedField mf, String algorithmDefault) {
             super();
             this.delegate = delegate;
             this.mf = mf;
+            this.algorithmDefault = algorithmDefault;
```

Comment:
Looks like this is no longer being used in this static inner class. Can this be removed from here?

Reply:
@cwperks Good spot. Removed.